SecretSanta/test/SecretSanta.Domain.Tests/Helpers/EnumerableRandomizeTests.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SecretSanta.Domain.Tests.Helpers
+{
+    [TestClass]
+    public class EnumerableRandomizeTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void EnumerableRandomize_ThrowsExceptionIfIRandomNull()
+        {
+            var list = Enumerable.Range(1, 5).ToList();
+            var randList = list.Randomize(null).ToList();
```

Comment:
Not a bad test but you could simplify it to just a single line.
```suggestion
            Enumerable.Empty<int>().Randomize(null);
```